Editor/EditorCore/PolyShapeTool.cs

```diff
@@ -0,0 +1,958 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using UnityEditor.EditorTools;
+using UnityEditor.ProBuilder.UI;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEngine.ProBuilder.MeshOperations;
+using Math = UnityEngine.ProBuilder.Math;
+using UObject = UnityEngine.Object;
+
+#if !UNITY_2020_2_OR_NEWER
+using ToolManager = UnityEditor.EditorTools.EditorTools;
+#else
+using ToolManager = UnityEditor.EditorTools.ToolManager;
+#endif
+
+
+namespace UnityEditor.ProBuilder
+{
+    public class PolyShapeTool : EditorTool
```

Comment:
Missing `[EditorTool("Edit PolyShape", typeof(PolyShape))]`

Reply:
I didn't add the attribute so that the tool does not appear in the custom tool menu but only in the PB Menu. 
Should this be modified?